tensorboard/webapp/tbdev_upload/tbdev_upload_button_component.ts

```diff
@@ -40,7 +45,14 @@ import {TbdevUploadDialogComponent} from './tbdev_upload_dialog_component';
   ],
 })
 export class TbdevUploadButtonComponent {
-  constructor(private dialog: MatDialog) {}
+  @HostBinding('class.shown') shown: boolean = false;
+
+  constructor(
+    @Inject('window') private readonly window: Window,
```

Comment:
after the change, the `window` object is no longer used so we can remove `private readonly` and just use it as a local variable.

Reply:
Done.